diagnostics/physics/sst_eval.py

```diff
@@ -101,24 +102,28 @@ def plot_sst_eval(pp_root):
     cbar2.ax.set_xlabel('SST difference (°C)')
     cbar2.ax.set_xticks([-2, -1, 0, 1, 2])
     grid[5].set_title('(e) Model - GLORYS12')
-    annotate_skill(model_ave, glorys_rg, grid[5], weights=model_grid.areacello)
+    annotate_skill(model_ave, glorys_rg, grid[5], weights=model_grid.areacello, x0=config['text_x'], y0=config['text_y'], yint=config['text_yint'])
 
     for ax in grid:
-        ax.set_xlim(-99, -35)
-        ax.set_ylim(4, 59)
+        ax.set_xlim(config['x']['min'], config['x']['max'])
+        ax.set_ylim(config['y']['min'], config['y']['max'])
         ax.set_xticks([])
         ax.set_yticks([])
         ax.set_xticklabels([])
         ax.set_yticklabels([])
         for s in ax.spines.values():
             s.set_visible(False)
         
-    plt.savefig('figures/sst_eval.png', dpi=300, bbox_inches='tight')
+    plt.savefig(config['figures_dir']+'sst_eval.png', dpi=300, bbox_inches='tight')
 
 
 if __name__ == '__main__':
     from argparse import ArgumentParser
+    import yaml
     parser = ArgumentParser()
-    parser.add_argument('pp_root', help='Path to postprocessed data (up to but not including /pp/)')
+    parser.add_argument('-p','--pp_root', type=str, help='Path to postprocessed data (up to but not including /pp/)')
+    parser.add_argument('-c','--config', type=str, help='Path to config.yaml file containing relevant paths for diagnostic scripts')
     args = parser.parse_args()
-    plot_sst_eval(args.pp_root)
+    with open(args.config, 'r') as f:
```

Comment:
@uwagura, I like the idea of using a YAML file to read the control parameters. I believe we can make this part as a function and move it to `plot_common.py`, so it can be reused by other scripts in the future:
Example in `plot_common.py`:
```
import logging
import yaml

# Configure logging for plot_common
logger = logging.getLogger(__name__)

def load_config(config_path: str):
    """Load the configuration file."""
    try:
        with open(config_path, 'r') as f:
            config = yaml.safe_load(f)
            logger.info(f"Loaded configuration from {config_path}")
        return config
    except Exception as e:
        logger.error(f"Error loading configuration from {config_path}: {e}")
        raise
```